src/certificator/certificator.py

```diff
@@ -14,6 +14,19 @@
 import base64
 import requests
 
+# Checks if an environment variable injected to F7T is a valid True value
+# var <- str
+# returns -> boolean
+def get_boolean_var(var):
+
+    # ensure variable to be a string
+    var = str(var)
+    # True, true or TRUE
+    # Yes, yes or YES
+    # 1
+
+    return var.upper() == "TRUE" or var.upper() == "YES" or var == "1"
+
```

Comment:
Why don't you import it from `cscs_api_common`, like in the other files?

Reply:
Because `certificator` needs to be built separately (for security reasons) and in order to easily build and deploy, all its dependencies are in the same file.